src/functions/TestDrive.ps1

```diff
@@ -24,39 +43,76 @@
                     TestDrivePath    = $testDrivePath
                 })
         }
-    } -EachBlockTearDownEnd {
+    }
+
+    $p.EachBlockTearDownEnd = {
         param($Context)
 
+        # Remap drive and variable if missing/wrong? Ex. if nested run was cancelled and didn't re-attach this drive
+        if (-not (& $script:SafeCommands['Test-Path'] TestDrive:\)) {
```

Comment:
Doing this at test teardown felt to excessive, though the test doesn't cost that much if we'd like to fix ASAP

Reply:
It does feel a bit excessive.